action.yml

```diff
@@ -36,6 +36,14 @@ inputs:
   settings-path:
     description: 'Path to where the settings.xml file will be written. Default is ~/.m2.'
     required: false
+  gpg-private-key:
+    description: 'Environment variable name for the GPG private key to import. Default is
```

Comment:
Is this really an environment variable name? [`README.md`](https://github.com/actions/setup-java/pull/59/files#diff-04c6e90faac2675aa89e2176d2eec7d8R116) appears to suggest it's not. So I'm just wondering.

Reply:
Good catch. It's not an environment variable.